docs/faq.md

```diff
@@ -0,0 +1,41 @@
+# FAQ
+1. What does it mean for an existing node operators to join Lido for Polygon? How do they transfer their slot?
+
+   1. Existing node operators can join without losing their slot and without downtime (take a look at the second scenario 
+      in the guide above). 
+   2. They will transfer their ownership to Lido for Polygon by calling the `joinOperator` function from the account that currently 
+   controls the validator. (this has to be done by using the same address that was used in stakeManager).
+
+2. How will their rewards and fees change? 
+   1. There will be the minimum of three reward income streams:
+      1. They will be able to collect all validator rewards (rewards accumulated from their stake).
+   
+      2. They will still receive the commission from the delegator rewards that they set. The commission rate is 
+         initially 5% (this value can be updated through the governance vote).
+      
+      3. Delegator rewards across all node operators are getting distributed in the following way: 2.5% to the dao  
+         2.5% goes to the treasury,  5% is distributed equally amongst all active node operators.
+
+3. What is the validator slashing? 
+   1. If the operator gets slashed by Polygon stakeManager, it will not receive any rewards until it became active again.
+      After slashing, the timestamp of the slashing is recorded, and is later used to check if the penalty period has passed. 
```

Comment:
Replace this second sentence with a link that redirects the user to polygon slashing docs.